ramalama/file.py

```diff
@@ -29,3 +31,44 @@ def __del__(self):
 
         if self.file:
             self.file.close()
+
+
+class PlainFile:
+
+    def __init__(self, filename: str, content: str = ""):
+        self.filename = filename
+        self.content = content
+
+    def write_to_file(self, dirpath: str):
+        with open(os.path.join(dirpath, self.filename), "w") as f:
+            f.write(self.content)
+            f.flush()
+
+    def from_file(dirpath: str, filename: str) -> "PlainFile":
+        file = PlainFile(filename)
+        with open(os.path.join(dirpath, filename), "r") as f:
+            file.content = f.read()
+        return file
+
+
+class IniFile:
+
+    def __init__(self, filename: str):
+        self.filename = filename
+        self.config = ConfigParser()
+        self.config.optionxform = lambda option: option
+
+    def add(self, section: str, key: str, value: str):
+        if section not in self.config:
+            self.config[section] = {}
+        self.config[section][key] = value
+
+    def write_to_file(self, dirpath: str):
```

Comment:
Simplify these function names to write and read?  I don't think you are going to have write_to_XYZ other then file.